core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -531,6 +531,34 @@ public static function findFirstPublishedRegularByPid($intPid, array $arrOptions
 		return static::findOneBy($arrColumns, $intPid, $arrOptions);
 	}
 
+	/**
+	 * Find the first published regular page by its parent ID
```

Comment:
Regular?

Reply:
fixed in dad9b92cf9e4ecdf0f44aef8ff65624953855320